lib/jekyll/tags/highlight.rb

```diff
@@ -110,13 +124,14 @@ def render_codehighlighter(code)
 
       def add_code_tag(code)
         code_attributes = [
-          "class=\"language-#{@lang.to_s.tr('+', '-')}\"",
+          "class=\"language-#{@lang.to_s.tr("+", "-")}\"",
```

Comment:
It's preferred to use `%Q{}` here, did it not warn about this? I don't know if Rubocop can but it it didn't we should probably check and see if it can, I'll admit though that this might be a limitation in static analysis (it should be though IMO.)